cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -1,3 +1,9 @@
+import config from '../../../support/config/services';
+
+const getParagraphText = blocks =>
+  blocks.find(el => el.type === 'paragraph' && el.markupType === 'plain_text')
```

Comment:
this will crash if nothing is found, but I guess it's a test file and we control the data so not much risk

Reply:
Yeah, basic expectation test assets have paragraphs, if they dont we wanna know :)